docs/concepts.md

```diff
@@ -211,7 +211,7 @@ Intuitively, data can only flow from `a` to `b` if:
 - `b` is **at least as secret** as `a`
 - `a` is **at least as trusted** as `b`
 
-The least privileged label is usually referred to as "public trusted" (and
+The least privileged label is usually referred to as "public untrusted" (and
 represented as `⊥`, pronounced "bottom"), which corresponds to a Node or Channel
```

Comment:
Just FYI this is very FLAM-specific. The more conventional thing in information flow would be for public_trusted to be the bottom of the lattice. Even in FLAM public_trusted is the bottom of the information flow lattice. public_untrusted is the bottom of the authority lattice which is a different thing.